client/buttonPanel/components/buttonPanel.js

```diff
@@ -7,23 +7,19 @@ import classNames from './scss/buttonPanel.module.scss'
 
 class ButtonPanel extends Component {
     start = () => {
-        const time = new Date().toString();
-        this.props.start({time});
+        this.props.startInit();
```

Comment:
no need in handlers anymore you can pass startInit and finishInit directly in render. And then make component functional